src/app/plugins/broadcast.plugin.ts

```diff
@@ -127,11 +127,20 @@ export class BroadcastPlugin extends Plugin {
   }
 
   private _reportToUser(message: IMessage) {
-    message.reply(
+    const content =
       `You are about to send:\n\`\`\`${this._ANNOUNCEMENT_CONTENT}\`\`\`\n` +
-        `to \`${this._CHANS_TO_SEND.length}\` classes... Are you sure?\n` +
-        `Respond with \`confirm\` or \`cancel\``
-    );
+      `to \`${this._CHANS_TO_SEND.length}\` classes... Are you sure?\n` +
+      `Respond with \`confirm\` or \`cancel\``;
+
+    this.container.messageService
+      .sendTextMessage(message, content, {
+        reply: true,
+      })
+      .catch((err) => {
+        this.container.loggerService.warn(
+          `Failed to ask for comfirmation on an announcement from ${message.author.username}. Error info:\n${err}`
+        );
+      });
```

Comment:
Not necessary. This plugin is usually sent to all classes so it just says `all` otherwise, just a select few of classes.

Reply:
It doesn't even list the classes, I put it in since you can make a long announcement.  Granted I have never seen one so I don't even know what you guys usually use it for.